cpp/models/abm/vaccine.h

```diff
@@ -33,33 +33,27 @@ namespace abm
  * Vaccine in ABM.
  * can be used as 0-based index
  */
-enum class Vaccine : std::uint32_t
+enum class ProtectionType : std::uint32_t
 {
-    Generic = 0,
-
+    NoProtection     = 0,
+    NaturalInfection = 1,
+    GenericVaccine   = 2, // Represent the Pfizer vaccine
```

Comment:
```suggestion
    GenericVaccine   = 2, 
```

Please remove comment here. Use it where you set the parameters.

Reply:
It is done.